packages/async-flow/src/replay-membrane.js

```diff
@@ -156,12 +156,13 @@ export const makeReplayMembrane = ({
       case 'promise': {
         const e = Error('where warning happened');
         console.log('Warning for now: vow expected, not promise', h, e);
+        throw E;
```

Comment:
Did you mean
```suggestion
        throw e;
```
?

After all, throwing `E` would be most strange!